bench/util/volumeRunner.js

```diff
@@ -0,0 +1,195 @@
+'use strict'
+// run this AFTER /database/loadVolumeData.js has populated db successfully
+const DEBUG = false // prints request/response details
+const START_SERVER = true // true = start udaru
+
+// ensure variables same as /database/loadVolumeData.js
+const NUM_TEAMS = 500 // total number of teams
+const TEAM_START_ID = 7 // user start id offset
+const NUM_USERS_PER_TEAM = 100
+const NUM_POLICIES_PER_TEAM = 10
+const DURATION = 15 // how long to run tests for in seconds
+
+const autocannon = require('autocannon')
+const path = require('path')
+
+// part of initial route, this changes for second run
+var partialRoute = '/authorization/access/'
+
+var child = null
+if (START_SERVER) {
+  const spawn = require('child_process')
+  // start udaru server
+  console.log('Starting UDARU server...')
+  // need stdin to determine when server starts
+  child = spawn.fork(path.join(__dirname, '../../lib/server/start'), [], { silent: true })
+
+  child.stdout.on('data', (m) => {
+    var s = m.toString('utf8')
+    if (s.indexOf('Server started on:') !== -1) {
```

Comment:
if the child process is logging heavily to stdout, this might create too much overhead.
A better approach might be to use use `process.send()`  and redirect stdout/stderr to `/dev/null`.

Reply:
thanks, made that change, had to change server/start to send the message when forked as child, hope it's OK